rust/src/address.rs

```diff
@@ -29,54 +29,100 @@ fn variable_nat_encode(mut num: u64) -> Vec<u8> {
 }
 
 #[derive(Debug, Clone, Eq, Ord, PartialEq, PartialOrd)]
-enum AddrCredType {
-    Key(Keyhash),
-    Script(Scripthash),
+enum StakeCredType {
```

Comment:
this was basically `stake_credential` in `shelley.cddl` so it was renamed and (de)serialization was implemented so we could call it from the generated code.